ci-operator/config/openshift/openshift-tests-private/.config.prowgen

```diff
@@ -62,6 +84,60 @@ slack_reporter:
                     {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
                     <{{.Status.URL}}|View logs>'
 
+- channel: '#managed-hypershift-ci-watcher'
+  excluded_variants: # It is possible to exclude notifications from specific variants, for example, when you don't require them from older releases
+  - some-old-release
+  job_names: # Listing of job names (ci-operator's 'as' field) that this configuration applies to
+  - aws-rosa-hcp-byo-kms-oidc-auditlog-int-full-f14
+  - aws-rosa-hcp-byo-kms-oidc-auditlog-int-full-f28
+  - aws-rosa-hcp-byo-kms-oidc-auditlog-stage-critical-f14
+  - aws-rosa-hcp-byo-kms-oidc-auditlog-stage-critical-f28
+  - aws-rosa-hcp-byo-kms-oidc-auditlog-stage-full-f14
+  - aws-rosa-hcp-byo-kms-oidc-auditlog-stage-full-f28
+  - aws-rosa-hcp-capi-private-stage-f7
+  - aws-rosa-hcp-capi-stage-f7
+  - aws-rosa-hcp-cilium-stage-full-f7
+  - aws-rosa-hcp-int-full-f14
+  - aws-rosa-hcp-int-full-f28
+  - aws-rosa-hcp-private-proxy-int-full-f14
+  - aws-rosa-hcp-private-proxy-int-full-f28
+  - aws-rosa-hcp-private-proxy-stage-critical-f14
+  - aws-rosa-hcp-private-proxy-stage-critical-f28
+  - aws-rosa-hcp-private-proxy-stage-full-f14
+  - aws-rosa-hcp-private-proxy-stage-full-f28
+  - aws-rosa-hcp-sector-advanced-prod-stable-critical-f28
+  - aws-rosa-hcp-sector-byo-kms-oidc-auditlog-prod-critical-f7
+  - aws-rosa-hcp-sector-byo-kms-oidc-auditlog-prod-critical-f28
+  - aws-rosa-hcp-sector-private-proxy-prod-critical-f7
+  - aws-rosa-hcp-sector-private-proxy-prod-critical-f28
+  - aws-rosa-hcp-sector-private-proxy-prod-stable-critical-f28
+  - aws-rosa-hcp-sector-prod-critical-f7
+  - aws-rosa-hcp-sector-prod-critical-f28
+  - aws-rosa-hcp-sector-prod-stable-critical-f28
+  - aws-rosa-hcp-stage-critical-f14
+  - aws-rosa-hcp-stage-critical-f28
+  - aws-rosa-hcp-stage-full-f14
+  - aws-rosa-hcp-stage-full-f28
+  job_states_to_report: #Accepts any ProwJob status
+  - error
+  - failure
+  - success
+  report_template: '{{if eq .Status.State "success"}} :rainbow: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :rainbow:
+                    {{else}} :volcano: Job *{{.Spec.Job}}* ended with *{{.Status.State}}*. <{{.Status.URL}}|View logs> :volcano: {{end}}'
+
+- channel: '#ocm-qe-prow-ci-jobs'
+  excluded_variants: # It is possible to exclude notifications from specific variants, for example, when you don't require them from older releases
+  - some-old-release
+  job_names: # Listing of job names (ci-operator's 'as' field) that this configuration applies to
+  - aws-rosa-sts-shared-vpc-f28
+  - aws-ocm-osd-ccs-f28
+  - aws-rosa-sts-f28
```

Comment:
The current channel for these three jobs needs to be corrected in Prow. All of them which run the OCP e2e test should be in `managed-hypershift-ci-watcher`. Only the jobs run the OCM testing are reported to `ocm-qe-prow-ci-jobs`. 

Please help update them acordingly. Thanks very much.

Reply:
Fix in commit [9665321](https://github.com/openshift/release/pull/57921/commits/9665321356b1c251d42f837586306d89e5e618a8)

Reply:
Thanks